Build failure on Redox with minimal versions

redox-syscall 0.2 cannot compile with recent nightly compilers.  It produces errors like this:
```
error[E0557]: feature has been removed
 --> /home/somers/.cargo/registry/src/github.com-1ecc6299db9ec823/redox_syscall-0.2.0/src/lib.rs:2:12
  |
2 | #![feature(const_fn)] // see https://github.com/rust-lang/rfcs/pull/2632
  |            ^^^^^^^^ feature has been removed
  |
  = note: split into finer-grained feature gates
```
Could you please raise the minimal version of the redox-syscall dependency to something that works?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,7 +33,7 @@ features = [
 ]
 
 [target.'cfg(target_os = "redox")'.dependencies]
-redox_syscall = "0.2"
+redox_syscall = "0.2.9"
 
 [dev-dependencies]
 doc-comment = "0.3"

